common/protob/messages-bitcoin.proto

```diff
@@ -534,4 +545,5 @@ message AuthorizeCoinJoin {
     repeated uint32 address_n = 4;                                      // prefix of the BIP-32 path leading to the account (m / purpose' / coin_type' / account')
     optional string coin_name = 5 [default='Bitcoin'];                  // coin to use
     optional InputScriptType script_type = 6 [default=SPENDADDRESS];    // used to distinguish between various address formats (non-segwit, segwit, etc.)
+    optional AmountUnit amount_unit = 11;                               // show amounts in
```

Comment:
let's set `[default=BITCOIN]` so that we don't need to deal with case this is not sent

Reply:
Fix in 3a1b98050463c515587349cf6abbb4090d6ee8ab